bode/bode/models/task_relation.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+
+from enum import Enum
+
+from sqlalchemy.dialects.postgresql import ENUM, UUID
+
+from bode.app import db
+
+
+class RelationType(Enum):
+    Dependent = "DEPENDENT"
+    Interchangable = "INTERCHANGABLE"
+    Subtask = "SUBTASK"
+
+    @classmethod
+    def list(cls):
+        return list(map(lambda c: c.value, cls))
+
+
+SYMMETRIC_RELATIONS = [RelationType.Interchangable.value]
+
+
+class TaskRelation(db.Model):
+    __tablename__ = "tasks_relations"
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    first_task_id = db.Column(UUID(as_uuid=True), db.ForeignKey("tasks.id"), nullable=False)
+    second_task_id = db.Column(UUID(as_uuid=True), db.ForeignKey("tasks.id"), nullable=False)
```

Comment:
If anyone comes up with better names for these columns pls let me know

Reply:
I think they are ok. I don't have any idea on how to describe them better